src/plugins/ini/ini.c

```diff
@@ -84,6 +84,8 @@ static int iniKeyToElektraKey (void *vhandle, const char *section, const char *n
 
 	keyAddBaseName (appendKey, name);
 
+	if(value == NULL)
+		keySetMeta(appendKey, "ini/empty", "");
```

Comment:
i couldn't really differentiate between 
`key = ` and `key` without the metakey when writing back to the config file